### DISCUSSION TITLE
Is there any way to have Copilot installed on 2 laptops?

### DISCUSSION THREAD

**Discussion by OB42** [community/community#18825] [Copilot Conversations] [answered]
Hi,
I have Copilot installed on my work laptop and personal laptop.
Every time I switch between the 2 computers I get disconnected from the other one. I use VS code or VS Code insiders, on ubuntu and Mac OS.
I guess it's a way to stop people from sharing their accounts? I have the same IP all the time on both laptops btw
Is there any way to circumvent this limitation? It's really annoying to have to chose between using it on just one of my laptops :/ or having to login again everytime
Thanks

**Comment by davecheney** [comment, marked as answer]
There is no limit on the number of computers that may be used concurrently providing you are within GitHub's acceptable use policy -- basically, don't share your github credentials with other people. 

I can see that you are running copilot/1.13.5700 in vscode on one of your systems, which would have prevented you using Copilot for well over a month. You should upgrade, and please stay up to date as we're rapidly releasing new versions of the plugin.
